src/DynamoPackages/PackageLoader.cs

```diff
@@ -386,9 +365,7 @@ public void LoadAll(LoadPackageParams loadPackageParams)
         /// <param name="packages"></param>
         public void LoadPackages(IEnumerable<Package> packages)
         {
-            var packageList = packages.ToList();
```

Comment:
I vaguely remember this was done to avoid some concurrency bug - can you check the history?

Reply:
This is the commit that added the `ToList`: https://github.com/DynamoDS/Dynamo/commit/8d9805e0b615bdb6a6d370723816c5293158b5a0#diff-7ae21a0e381958737cc115ada84cbd9f96efe3315928cbc44b367967923cf55b
I still don't see a reason to do that.